common/csv_utils.go

```diff
@@ -18,3 +18,25 @@ func ReadCSV(path string) ([][]string, error) {
 
 	return csv.NewReader(csvFile).ReadAll()
 }
+
+func SaveCSV(path string, records [][]string) {
+	if _, err := os.Stat(path); os.IsExist(err) {
+		os.Remove(path)
+	}
+
+	file, err := os.Create(path)
+	checkError("Cannot create file: ", err)
```

Comment:
Every Go dev knows how the `if err != nil` can clutter everything, but it's ok to have that all over the place, that's how Go is designed, you don't need to create a function to check if the error is nil